DEVELOPMENT.md

```diff
@@ -35,15 +35,14 @@ To build images only, use `docker-compose build`
 
 ### Developing in Supabase Mode
 
-To run the dev env in Supabase mode, adjust the `.docker.env` config:
+To run the dev env in Supabase mode use the following command:
 
 ```
-LOGFLARE_SINGLE_TENANT=true LOGFLARE_SUPABASE_MODE=true
```

Comment:
should separate out the two:
- supabase mode uses needs both flag
- single tenant mode (without the data seeding) is only the single tenant flag. maybe can just add the env var to the .docker.env file commented out.